NextExample/babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
I wonder if we should use the reanimated SWC plug-in instead of Babel? Or perhaps do a run with and a run without it, using an env variable?

For reference: https://github.com/showtime-xyz/showtime-frontend/blob/staging/apps/next/next.config.js#L91-L99

This isn’t a requirement for this PR, but would be useful for testing.

Reply:
Good idea, we can test the SWC plugin as well once it's merged (in a separate PR).